File: CS/WindowsApplication1/MyMemoExEdit/MyMemoExEdit.cs
using System;
using System.ComponentModel;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.Registrator;
using DevExpress.XtraEditors.Controls;
using DevExpress.Utils.Drawing;
using DevExpress.XtraEditors.ViewInfo;

namespace WindowsApplication1
{

    public class MyMemoExEdit : MemoExEdit
    {
        static MyMemoExEdit()
        {
            RepositoryItemMyMemoExEdit.Register();
        }
     
        public override string EditorTypeName
        {
            get { return RepositoryItemMyMemoExEdit.EditorName; }
        }
        [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
        public new RepositoryItemMyMemoExEdit Properties
        {
            get { return base.Properties as RepositoryItemMyMemoExEdit; }
        }
    }
}

File: VB/WindowsApplication1/MyMemoExEdit/MyMemoExEdit.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.Registrator
Imports DevExpress.XtraEditors.Controls
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraEditors.ViewInfo

Namespace WindowsApplication1

	Public Class MyMemoExEdit
		Inherits MemoExEdit
        Shared Sub New()
            RepositoryItemMyMemoExEdit.Register()
        End Sub

		Public Overrides ReadOnly Property EditorTypeName() As String
			Get
				Return RepositoryItemMyMemoExEdit.EditorName
			End Get
		End Property
		<DesignerSerializationVisibility(DesignerSerializationVisibility.Content)> _
		Public Shadows ReadOnly Property Properties() As RepositoryItemMyMemoExEdit
			Get
				Return TryCast(MyBase.Properties, RepositoryItemMyMemoExEdit)
			End Get
		End Property
    End Class
End Namespace
